### PR TITLE
go: variable name for controllercontext

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: go/variable_names.md [hunk 1]
```diff
@@ -5,11 +5,24 @@ tend to use in our code. Aligning with the suggestions below makes our code
...
 
-In our `operatorkit` resource implementations, we take the parameter `obj`
-and have to assert the proper type the resource actually works with. The
-variable name we want to use here is `cr`.
+In `operatorkit` resource implementations, we sometimes work with
+`controllercontext` package. We store data extracted from this context in
+variable named `cctx`.
+
+```go
+cctx, err := controllercontext.FromContext(ctx)
```
Comment (kopiczko) [inline go/variable_names.md]: If `cctx` is the best name is debatable. Let's agree on something and start using it.

So far some call it:

- `cc` - I find it confusing with cloudconfig which we often assign to `cc` in our operator. Cloudconfig is a thing of the past though. So fine with me.
- `ctlCtx` - This is ok imo, but cumbersome to type.

So `cctx` is a mixture of both I just made up.

File: go/variable_names.md [hunk 2]
```diff
@@ -5,11 +5,24 @@ tend to use in our code. Aligning with the suggestions below makes our code
...
+if err != nil {
+	creturn nil, microerror.Mask(err)
+}
+```
+
+## Custom Resource
+
+In `operatorkit` resource implementations, we take the parameter `obj` and have
```
Comment (kopiczko) [inline go/variable_names.md]: I replaced ```In our `operatorkit` ``` with ```In `operatorkit` ``` and git thinks I changed whole thing.

File: go/variable_names.md (hunk 1 above)
Comment (rossf7) [reply go/variable_names.md]: My vote is for `ctlCtx`. It is longer but its clearer for me. `cctx` looks a bit like a typo to me.

**Review by rossf7 — APPROVED** [review]
LGTM and left my vote for the naming question :)

File: go/variable_names.md (hunk 1 above)
Comment (xh3b4sd) [reply go/variable_names.md]: My vote goes on `cc`.

Comment (TheoBrigitte) [reply go/variable_names.md]: I prefer `ctlCtx` as it avoid any colision in case we already have `cc` cloudconfig variable.

**Review by TheoBrigitte — APPROVED** [review]

File: go/variable_names.md (hunk 1 above)
Comment (kopiczko) [reply go/variable_names.md]: When we don't like `cctx` and I looked at the code lately I prefer `cc`. We are even now. What should I do? :D

Comment (xh3b4sd) [reply go/variable_names.md]: When nobody vetos `cc` until tomorrow we go with it.

Comment (MarcelMue) [reply go/variable_names.md]: I vote for `cc` let's get this over with.

**Comment by kopiczko** [comment]
@MarcelMue decided it's `cc`. Merging.

**Pull request merged**